### PR TITLE
chore: add schema option

### PR DESCRIPTION
**Pull request by matex1024**

### CODE REVIEW

File: src/dialects/base-database.parser.ts [hunk 1]
```diff
@@ -42,6 +42,10 @@ export class BaseDatabaseParser {
     throw new Error('Implement "parse" method for your database parser!');
   }
 
+  public async getSchemas(): Promise<string[]> {
+    throw new Error('Implement "getSchema" method for your database parser!');
```
Comment (dziraf) [inline src/dialects/base-database.parser.ts:46]: `getSchema` or `getSchemas`?

File: src/dialects/postgres.parser.ts [hunk 2]
```diff
@@ -73,6 +73,7 @@ export class PostgresParser extends BaseDatabaseParser {
   public static dialects = ['postgresql' as const];
 
   public async parse() {
+    const schemaName = await this.getSchemas();
```
Comment (dziraf) [inline src/dialects/postgres.parser.ts:76]: `'schemaName' is assigned a value but never used`

File: src/dialects/postgres.parser.ts [hunk 3]
```diff
@@ -85,11 +86,10 @@ export class PostgresParser extends BaseDatabaseParser {
     return new DatabaseMetadata(this.connectionOptions.database, resourceMap);
   }
 
-  public async getTables() {
-    const query = await this.knex.raw(`
+  public async getTables(schema = 'public') {
```
Comment (dziraf) [inline src/dialects/postgres.parser.ts:89]: why `schema` arg, shouldn't it just use `this.schemaName`?

**Review by dziraf — CHANGES_REQUESTED** [review]
@matex1024 looks like build is failing in CI

**Comment by devjiwonchoi** [comment]
Hi, I'm the one who brought up this issue and realized that it hasn't been resolved. Regardless of the steps of contribution, if you all don't mind may I participate to solve this issue?

**Pull request closed without merging**